community/portworx/templates/portworx-storageclasses.yaml

```
@@ -32,7 +32,7 @@ parameters:
    io_profile: "db"
 ---
 kind: StorageClass
-apiVersion: storage/v1
+apiVersion: storage.k8s.io/v1beta1
```

This should be the GA version of this spec:
```suggestion
apiVersion: storage.k8s.io/v1
```